File: frontend/src/Types/team.ts
import type { Guid } from './api';

export interface Team {
  id: Guid;
  name: string;
  memberIds: Guid[];
  sprintIds: Guid[];
}

export interface CreateTeamRequest {
  name: string;
}

export interface UpdateTeamRequest {
  name: string;
}

File: backend/src/ScrumLog.Api/Data/Entities/Team.cs
namespace ScrumLog.Api.Data.Entities;

public class Team
{
    public Guid Id { get; set; }

    public required string Name { get; set; }

    public ICollection<Person> Members { get; set; } = new List<Person>();

    public ICollection<Sprint> Sprints { get; set; } = new List<Sprint>();
}
